package.json

```diff
@@ -73,6 +73,7 @@
     "dotenv": "^10.0.0",
     "flat": "^5.0.2",
     "node-vault": "^0.9.22",
+    "secretsfoundry": "^0.1.15",
```

Comment:
This seems wrong. Why should secretsfoundry be a dependency for itself?

Reply:
This was added by running this command `npm install secretsfoundry --save ` from readme. I will remove it